tmt/utils.py

```diff
@@ -171,8 +171,84 @@ def unrooted(self) -> 'Path':
 # A stand-in variable for generic use.
 T = TypeVar('T')
 
-# A FMF context type, representing name/values context.
-FmfContextType = Dict[str, List[str]]
+
+class FmfContext(Dict[str, List[str]]):
+    """
+    Represents a fmf context.
+
+    See https://tmt.readthedocs.io/en/stable/spec/context.html
```

Comment:
```suggestion
    See https://tmt.readthedocs.io/en/latest/spec/context.html
```

Sorry, obsessed with consistency ;-)

Reply:
Fixed in 76e82ed9